asyncscope.md

```diff
@@ -2249,31 +2335,25 @@ concept async_scope_association =
     requires(const Assoc& assoc) {
         { static_cast<bool>(assoc) } noexcept;
     };
-}
-```
-[2]{.pnum} `async_scope_association<Assoc>` is modeled only if `Assoc`'s copy and move operations are not potentially
-throwing.
-
-__`std::execution::async_scope_token` [exec.asyncscopetoken.concept]__
-
-[1]{.pnum} The `async_scope_token<Token>` concept defines the requirements on an object of type `Token` that can
-be used to create associations between senders and an async scope.
-```cpp
-namespace std::execution {
 
-template <class Token>
-concept async_scope_token =
+template <class Token, class Sender>
+concept async_scope_token_for =
```

Comment:
Looks like I mistakenly changed `async_scope_token<Token>` to `async_scope_token_for<Token, Sender>` in my PR. Oops.
```suggestion
template <class Token>
concept async_scope_token =
```